rotkehlchen/chain/avalanche/manager.py

```diff
@@ -107,40 +107,34 @@ def get_code(self, account: ChecksumEthAddress) -> str:
 
     def get_transaction_receipt(
             self,
-            tx_hash: str,
+            tx_hash: EVMTxHash,
     ) -> Dict[str, Any]:
         tx_receipt = self.covalent.get_transaction_receipt(tx_hash)
         if tx_receipt is None:
             tx_receipt = self.w3.eth.get_transaction(tx_hash).__dict__  # type: ignore
+            tx_receipt['hash'] = EVMTxHash(tx_receipt['hash'])  # web3 returns HexBytes
             return tx_receipt
         try:
-            # Turn hex numbers to int
-            tx_receipt.pop('from_address_label', None)
-            tx_receipt.pop('to_address_label', None)
-            block_number = tx_receipt['block_height']
-            tx_receipt['blockNumber'] = tx_receipt.pop('block_height', None)
-            tx_receipt['cumulativeGasUsed'] = tx_receipt.pop('gas_spent', None)
-            tx_receipt['gasUsed'] = tx_receipt['cumulativeGasUsed']
-            successful = tx_receipt.pop('successful', None)
-            tx_receipt['status'] = 1 if successful else 0
-            tx_receipt['transactionIndex'] = 0
```

Comment:
Why are you not including the status and transactionIndex? We do include them in the ethereum receipts.

We should strive for the result to be consistent.